packages/core/database/lib/schema/schema.js

```diff
@@ -105,7 +105,7 @@ const getColumnType = attribute => {
     case 'increments': {
       return {
         type: 'increments',
-        args: [{ primary: true }],
+        args: [{ primary: true, primaryKey: true }],
```

Comment:
Are `primary` and `primaryKey` redundant? Can we remove `primary` ?

Reply:
Nope we can't it's for retro compatibility :)

Reply:
I see, didn't think of that :) Thank